automation.py

```diff
@@ -143,15 +143,38 @@ def runOnProviders(self, func):
             func(v)
 
     def _validate(self, config_dictionary):
+        # In this function we have not set up our robust logging facilities yet
```

Comment:
I'm not sure what this means, is it that we haven't moved from our basic logger to the full loggingprovider yet?